keyboards/maxr1998/phoebe/info.json

```
@@ -0,0 +1,73 @@
+{
+  "keyboard_name": "Phoebe",
+  "keyboard_folder": "maxr1998/phoebe",
+  "url": "https://github.com/Maxr1998/Phoebe_Keyboard",
+  "maintainer": "Maxr1998",
+  "width": 12,
+  "height": 5,
+  "layouts": {
+    "LAYOUT_phoebe": {
```

The name here needs to be a case-sensitive match to the macro name in `phoebe.h` for this JSON object to work.

```suggestion
    "LAYOUT": {
```